### ISSUE TITLE
Infinite Wait

### ISSUE THREAD

**Issue by wangjoshuah** [minodisk/bigquery-runner#36]
Hi. I've been using the bigquery runner extension for some time now, and recently, when I try to query for data, the spinner just keeps looping in the toolbar and nothing ever appears.

I can see a list of datasets and tables in the `BIGQUERY RUNNER: RESOURCES` tab in VSCode. But I cannot preview or query for any data. The spinner just keeps looping (never timing out). I've tried logging out and logging back in, but there has been no change. 

Here are my vscode settings
```
{
 "bigqueryRunner.keyFilename": "XXX/api_key.json",
  "bigqueryRunner.downloader.csv.header": true,
  "[sql-bigquery]": {
    "editor.defaultFormatter": "m-ahmadkhan.bq-formatter-vscode"
  },
  "bq-formatter.newlineBeforeSemicolon": true,
  "bq-formatter.useTabs": true,
  "bq-formatter.indentStyle": "tabularRight",
  "notebook.output.textLineLimit": 50,
  "bigqueryRunner.defaultDataset.datasetId": ""
}
```

Any idea why there might be an infinite loop?

**Comment by waseemkhalifa** [comment]
I have having the same issue also, i think this might be related to a new update to vscode.

**Comment by minodisk** [comment]
Sorry I haven't worked on it, I'll take a look after work today.

**Comment by davidchung-hifranc** [comment]
Same issue here, no warnings or bug reports. The query just runs forever with no timeout. Query works when run on Bigquery

**Comment by waseemkhalifa** [comment]
> I have having the same issue also, i think this might be related to a new update to vscode.

Just to confirm, I've reverted my vscode to the version prior to the latest update, and the bigquery extension is working as expected. So it's related to the new update to vscode imo

**Comment by minodisk** [comment]
@wangjoshuah @waseemkhalifa @davidchung-hifranc

I'm really sorry and thanks for debugging this. I found the cause and I think it's because the extension was using a very old URI scheme that was no longer supported in the latest version. I'll release a fixed version tonight.

**Comment by waseemkhalifa** [comment]
@minodisk thank you! Really appreciate your work on this extension, it's brilliant, I use it daily at work!

**Comment by minodisk** [comment]
Fixed in the now released v1.21.0.
I have confirmed that the combination of VS Code v1.73.0 and BigQuery Runner v1.21.0 works in my environment, but if you find problems in yours, please let me know.
